grr/server/grr_response_server/gui/ui/store/config_facade.ts

```diff
@@ -55,6 +62,14 @@ export class ConfigStore extends ComponentStore<ConfigState> {
           }),
           ));
 
+  private readonly fetchClientsLabels = this.effect<void>(
+      obs$ => obs$.pipe(
+          switchMapTo(this.httpApiService.fetchAllClientsLabels()),
+          map(apiClientsLabels =>
+              apiClientsLabels.map(getApiClientLabelName)),
```

Comment:
Please add a short comment why you only get the label name (stating that the API includes `owner` inconsistently).